Allow snapping without animation

derrh opened the pull request:
My use case is that I need to update an intractable view's position while it is just off screen (in a paging scroll view) and I don't want it to be mid snap animation if the user swipes over to the page containing the intractable view.

This is very likely a naive approach. Also, I have only tested this on iOS, since my current project is iOS only right now.

ButtersHub commented:
Thanks for the two PRs, we will review the code and merge it.

kptlronyttcna commented on android/src/main/java/com/wix/interactable/InteractableViewManager.java:
```diff
@@ -51,8 +51,13 @@ public void receiveCommand(
+                ReadableMap params = args.getMap(0);
+                int snapPoint = params.getInt("index");
+                boolean animated = true;
+                if (params.hasKey("animated") {
```
if (params.hasKey("animated")) {

matpaul commented:
@ButtersHub Any updates on this PR ?)

derrh commented:
This could be a useful feature, but I was able to find a more elegant solution the problem I was solving. Feel free to close it if you don't see value in snapping without animation. Also, the android code is not tested. I addressed feedback from @kptlronyttcna, but I haven't tested the code on Android.

silyevsk commented:
Thank you @derrh ! In the end we've added it separately in changePosition function a while ago (https://github.com/wix/react-native-interactable/issues/60)
